config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -109,6 +109,19 @@ spec:
       secret: osp-secret
   nova:
     template:
+      cellTemplates:
+        cell0:
+          cellDatabaseAccount: nova-cell0
+          conductorServiceTemplate:
+            replicas: 1
+          hasAPIAccess: true
+        cell1:
+          cellDatabaseAccount: nova-cell1
+          cellDatabaseInstance: openstack-cell1
+          cellMessageBusInstance: rabbitmq-cell1
+          conductorServiceTemplate:
+            replicas: 1
+          hasAPIAccess: true
       secret: osp-secret
```

Comment:
i belive we coudl also just document 

cellTemplates:
  cell0:{}
  cell1:{}

but i want to show specifying the db and message bus explicitly
which actual speaking of which i am missing both in my cell0 template
ill fix that and respin